src/read_from_pdf.py

```diff
@@ -0,0 +1,103 @@
+import os
+import re
+import argparse
+import subprocess
+import numpy as np
+
+from chunker import try_to_write_dir
+
+def filter_junk_text(txt_file: str, output_dir: str):
+    # helping file to get content of txt file
+    filtered_document_text = ''
+    document_content = open(txt_file, 'r')
+
+    # check in every line of the text if there is junk text there
+    for line in document_content:
+        line = remove_line_if_junk(line)
+        filtered_document_text += line
+
+    # write filtered text into new directory
+    file_name_index = txt_file.rfind('/')
+    txt_file = txt_file[file_name_index+1:]
+
+    f = open(output_dir + txt_file, 'w')
+    f.write(filtered_document_text)
+    f.close()
+
+def remove_line_if_junk(line: str) -> str:
+    junk_list_document = ['\s[nN]ichtamtliche Lesefassung.*']
+
+    # check if the line includes any element of the list
+    for i in junk_list_document:
+        x = re.search(i, line)
+
+        if x:
+            # remove the found text
+            # print(x[0]) # Debug
+            line = re.sub(x[0], '', line)
+        
+    return line
+
+def main():
+    """Main to translate given PDF files after argument '-c' in a specific folder 
+    into txt files that retain the layout, e.g. table
+    """
+
+    parser = argparse.ArgumentParser(description = 'Translates all PDF files' +
+                        ' from a given folder into text files while' +
+                        ' retaining the layout including table structures.' +
+                        ' The new text files will be located in the same' +
+                        '  directory as the PDF files.')
+
+    parser.add_argument('-d', '--dir_path', type = try_to_write_dir, 
+                       default = './',
+                       help = 'Directory in which all PDF files are saved.')
```

Comment:
What about text files?

Reply:
Does the program save any pdf files at all?

Reply:
When it is converted from pdf to txt the text files are in the same directory as pdf. When they are filtered, they are saved in the directory given with the argument ´-o´

The program does not save any pdf files. They should be provided by the user in the appropriate directory.

Reply:
The help text from line 58 reads as if this programs is supposed to save PDF files. Please change this, e.g. replace PDF by text.